digiwf-apps/packages/apps/digiwf-tasklist/src/views/TaskDetail.vue

```diff
@@ -189,144 +199,88 @@ export default class TaskDetail extends SaveLeaveMixin {
   isCompleting = false;
   hasCompleteError = false;
   isCancelling = false;
-  hasCancelError = false;
-  isCancelable = false;
+  hasCancelError = false
   cancelText = "Aufgabe Abbrechen";
 
   isDownloading = false;
   hasDownloadError = false;
   hasDownloadButton = false;
   downloadButtonText = "Dokument herunterladen";
 
+  /**
+   * FIXME: Is it only the variable for showing or hidding the dialog?
+   */
   followUp = false;
 
+  /**
+   * toggle for showing fab menu
+   */
   fab = false;
 
   @Prop()
   id!: string;
 
   @Provide('formContext')
-  get formContext(): FormContext { return {id: this.id, type: "task"}};
+  get formContext(): FormContext {
+    return {id: this.id, type: "task"}
+  };
 
   @Provide('apiEndpoint')
-  apiEndpoint = EngineServiceApiConfig.base;
+  apiEndpoint = ApiConfig.base;
 
   created() {
-    this.loadTask();
+    console.log("created")
+    loadTask(this.id).then(({data, error}) => {
+      if (!!data) {
+        this.task = data.task;
+        this.model = data.model;
+        this.followUpDate = data.followUpDate;
+        this.cancelText = data.cancelText
+        this.hasDownloadButton = data.hasDownloadButton;
+        this.downloadButtonText = data.downloadButtonText;
+      }
+      if (!!error) {
+        this.errorMessage = error;
+      }
+    });
   }
 
   mounted() {
     // Apply a @click.stop to the .v-speed-dial__list that wraps the default slot
-    this.$el
-      .querySelector(".v-speed-dial__list")!
-      .addEventListener("click", (e) => {
-        e.stopPropagation();
-      });
+    // this.$el
+    //   .querySelector(".v-speed-dial__list")!
+    //   .addEventListener("click", (e) => {
+    //     e.stopPropagation();
+    //   });
   }
 
-  async completeTask(model: any): Promise<void> {
+  completeTask(model: any) {
```

Comment:
Der Request ist async und wird nicht mehr abgewartet, daher kann `async` hier entfernt werden, richtig?

Reply:
An sich können completes ja etwas länger laufen und UI muss verfügbar bleiben

Reply:
doch. der request wird noch abgewartet. in dem fall wird das dann in dem .then block des promises gehandelt. 
eine funktion muss man nur dann als async markeren, wenn man in dieser ein await verwendet.

Reply:
Ich glaub wir verstehen uns hier falsch. 😅

Ich wollte hier nur sichergehen, dass das async Verhalten sich hier nicht ändert. Das die Logik im Callback auf den Request wartet ist klar. Vorher war die Methode Async und man hat mit await auf den Request gewartet, nun ist die Methode nicht mehr Async und die Logik ist in den Callback gewandert, womit sich das Verahtlen aber nicht ändert.

Reply:
Kannst resolven, wenn das so ist und ich nichts übersehen habe